tokio-trace/src/lib.rs

```diff
@@ -309,10 +309,12 @@
 //! [`tokio-trace-futures`]: https://github.com/tokio-rs/tokio-trace-nursery/tree/master/tokio-trace-futures
 //! [`tokio-trace-fmt`]: https://github.com/tokio-rs/tokio-trace-nursery/tree/master/tokio-trace-fmt
 //! [`tokio-trace-log`]: https://github.com/tokio-rs/tokio-trace-nursery/tree/master/tokio-trace-log
+extern crate cfg_if;
 extern crate tokio_trace_core;
 
 // Somehow this `use` statement is necessary for us to re-export the `core`
 // macros on Rust 1.26.0. I'm not sure how this makes it work, but it does.
+use cfg_if::cfg_if;
```

Comment:
Importing macros this way (with the `use` keyword) doesn't work on Rust 1.26, `tokio`'s minimum supported Rust version (this is why your CI build is failing).

We'll need to import the macro the "old-fashioned" way instead, by putting the `#[macro_use]` attribute on the `extern crate` statement.